public/docs/ts/latest/guide/i18n.jade

```diff
@@ -129,7 +129,7 @@ a(id="i18n-plural-pipe")
 
 .alert.is-important
   :marked
-    The ng-xi18n messages extractor doesn't support the plural pipe right now.
+    The ng-xi18n messages extractor doesn't support the plural pipe.
```

Comment:
`<div title="{count, plural, ...}">` will be transformed to `<div title="count|i18nPlural(...)">` before compilation.

extraction:
- `{count, plural, ...}` will be extracted by `ng-xi18n`,

merge:
- `ngc` will merge the translated ICU message,
- then ICU messages will be converted to pipes,
- then the template is compiled.